cardano-db-sync/src/Cardano/DbSync/OffChain.hs

```diff
@@ -0,0 +1,269 @@
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE NumericUnderscores #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE Rank2Types #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE NoImplicitPrelude #-}
```

Comment:
I notice we have a lot of modules using both Prelude and Cardano.Prelude. Should we use this pattern everywhere (`NoImplicitPrelude`)?

Reply:
could set it globally in the cabal file? unless we need `Prelude` for some reason